### PR TITLE
Added a test for daily_min

### PR DESCRIPTION
**Pull request by Monarda**
Here's some code Patrick, I was too lazy to test it!

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -29,3 +29,17 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+def test_daily_min_integers():
+    """Test that mean function works for an array of positive integers."""
```
Comment (patrick-austin) [inline tests/test_models.py]: Docstring refers to mean, not min

File: tests/test_models.py [hunk 2]
```diff
@@ -29,3 +29,17 @@ def test_daily_mean_integers():
...
+    from inflammation.models import daily_mean
```
Comment (patrick-austin) [inline tests/test_models.py]:
```suggestion
    from inflammation.models import daily_min
```
Importing wrong function

File: tests/test_models.py [hunk 3]
```diff
@@ -29,3 +29,17 @@ def test_daily_mean_integers():
...
+
+    test_input = np.array([[1, 2],
+                           [3, 4],
+                           [5, 6]])
+    test_result = np.array((1,2))
+    test_result[0] = 1
+    test_result[1] = 4
```
Comment (patrick-austin) [inline tests/test_models.py]:
```suggestion
    test_result = np.array([1,2])
```
Simpler assignment and correcting min value for column 2.

**Review by patrick-austin — CHANGES_REQUESTED** [review]
There are some errors with the test.

File: tests/test_models.py (hunk 3 above)
Comment (Monarda) [reply tests/test_models.py]: Yes, should be 2 not 4. But clearly my method of working with numpy is right!

Comment (patrick-austin) [reply tests/test_models.py]: OK, you know best!

**Review by patrick-austin — APPROVED** [review]
No more errors, I think we can merge this code.

**Pull request merged**